lib/nuorder.rb

```diff
@@ -5,23 +5,10 @@
 
 module Nuorder
   class ApiError < StandardError; end
-  extend Nuorder::Configurable
 
   class << self
-    def client
```

Comment:
I don't think there's any need to get rid of the global shared client. It's convenient and a common pattern when thread safety isn't a concern.

We just need to remove the sharing of state between the global client and other clients.